frontend/src/routes/smartgoals/+page.svelte

```diff
@@ -1,30 +1,141 @@
-<div class="card flex-shrink-0 w-full lg:w-1/3 max-w-sm shadow-2xl bg-base-100">
-  <div class="card-body">
-    <div class="form-control">
-      <label class="label" for="email-input">
-        <span class="label-text">Email</span>
-      </label>
-      <input type="text" placeholder="email" id="email-input" class="input input-bordered" />
-    </div>
+<script>
```

Comment:
```suggestion
<script lang="ts">
```

Reply:
That fixed many of the errors.

Reply:
Okay. Scratch that, I saved the file and errors popped back up.